packages/core/server/index.js

```diff
@@ -38,7 +37,7 @@ const passthrough = createProxyMiddleware({
   changeOrigin: true,
   followRedirects: true,
   secure: 'development' !== process.env.NODE_ENV,
-  target: API_ORIGIN || API_ROOT_URL.replace('/wp-json/irving/v1', ''),
+  target: process.env.API_ORIGIN || process.env.API_ROOT_URL.replace('/wp-json/irving/v1', ''),
```

Comment:
any benefits to removing the destructuring here?

Reply:
Honestly, not sure. But it was complaining the variable didn't exist when I was testing locally. Like it couldn't find it.

Reply:
Gotcha - not a blocker! Destructure defaults might be able to fix the eslint warning though